### DISCUSSION TITLE
html template with custom post type

### DISCUSSION THREAD

**Discussion by kodaizen** [WordPress/gutenberg#40152] [Q&A] [answered]
Hello, how can I use an html file from the templates folder to display a custom post type (archive & single)?

**Comment by t-hamano** [comment, marked as answer]
Following the rules of the classic theme, I think you should only need to place HTML files in the `templates` folder.

For example, `templates/single-book.html`, `templates/archive-book.html`, etc.

**Comment by kodaizen** [comment]
ok, thank you!
